volumegroup/README.md

```diff
@@ -0,0 +1,331 @@
+}
+// DeleteVolumeGroupResponse holds the information to send when
+// volumeGroup is successfully deleted.
+message DeleteVolumeGroupResponse {
+}
+```
+
+##### DeleteVolumeGroup Errors
+
+If the plugin is unable to complete the DeleteVolumeGroup call successfully, it MUST return a non-ok gRPC code in the
+gRPC status. If the conditions defined below are encountered, the plugin MUST return the specified gRPC error code. The
+CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
+
+| Condition           | gRPC Code             | Description                                                                                                                                                                                                                | Recovery Behavior                                                                                                                                     |
+| ------------------- | --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Volume group in use | 9 FAILED_PRECONDITION | Indicates that the volume group corresponding to the specified `volume_group_id` could not be deleted because it is in use by another resource or has snapshots and the plugin doesn't treat them as independent entities. | Caller SHOULD ensure that there are no other resources using the volume group and that it has no snapshots, and then retry with exponential back off. |
+
+#### `ModifyVolumeGroupMembership`
+
+This RPC will be called by the CO to modify an existing volume group on behalf of a user. volume_ids provided in the
+ModifyVolumeGroupMembershipRequest will be compared to the ones in the existing volume group. New volume_ids in the
+modified volume group will be added to the volume group. Existing volume_ids not in the modified volume group will be
+removed from the volume group. If volume_ids is empty, the volume group will be removed of all existing volumes. This
+operation MUST be idempotent.
+
+File-based storage systems usually do not support this PRC. Block-based storage systems usually support this PRC.
+
+By adding an existing volume to a group, however, there is no way to pass in parameters to influence placement when
+provisioning a volume.
+
+It is out of the scope of the CSI spec to determine whether a group is consistent or not. It is up to the storage
+provider to clarify that in the vendor specific documentation. This is true either when creating a new volume with a
+group id or adding an existing volume to a group.
+
+CSI drivers supporting MODIFY_VOLUME_GROUP_MEMBERSHIP MUST implement ModifyVolumeGroupMembership RPC.
```

Comment:
Where is `MODIFY_VOLUME_GROUP_MEMBERSHIP` defined? I would expect it to be in the identity service, but have not seen it there. This also counts for `LIST_VOLUME_GROUPS`, and maybe others?

Reply:
done added capabilities for all of them

Reply:
Thanks!